environments/vm/secrets/all.yml

```diff
@@ -112,6 +112,10 @@ client_idp_users:
 google_client_id: "changethisclientid.apps.googleusercontent.com"
 google_client_secret: "CHANGETHISSECRET"
 
+# ORCID Backend
+orcid_client_id: "APP-0JWS2QIZ7UI6WXGR"
+orcid_client_secret: "8fd6f167-80e6-4433-8f26-60179897d2fc"
```

Comment:
Zijn dit waarden die we vrij kunnen gebruiken in de vm?  Moeten we dan andere hebben voor test ed?

Reply:
Maar moet deze credential hier wel staan dan?

Reply:
Tsja, ze doen het niet, maar een placeholder is inderdaad misschien mooier?